plugins/hydra_optuna_sweeper/tests/test_optuna_sweeper_plugin.py

```diff
@@ -29,6 +30,18 @@ def test_discovery() -> None:
     ]
 
 
+def check_distribution(expected: BaseDistribution, actual: BaseDistribution) -> None:
+    if not isinstance(expected, CategoricalDistribution):
+        assert expected == actual
+        return
+
+    assert isinstance(actual, CategoricalDistribution)
+    # shuffle() will randomize the order of items in choices.
+    assert list(sorted(actual.choices, key=lambda t: str(t))) == list(
```

Comment:
I didnt get it. Isnt the left side and the right side exactly same?

Reply:
Maybe you meant to use `expected` on one of the sides

Reply:
Ahh nvm you fixed it in a later commit

Reply:
Sorry for my carelessness. It was fixed in the commit https://github.com/facebookresearch/hydra/pull/1132/commits/faf96fd58707928f058c0a3311e58daf10e6cef0. I also simplified the assertion (thanks for your suggestion, @himkt).